core/state/state_object.go

```diff
@@ -574,7 +574,7 @@ func (s *stateObject) CodeSize() int {
 }
 
 func (s *stateObject) SetCode(codeHash common.Hash, code []byte) {
-	s.db.journal.SetCode(s.address)
+	s.db.journal.codeChange(s.address)
```

Comment:
Hm, I'd prefer to just change case, but leave it at `setCod`. This operation sets the code, whereas `codeChange` can imply that the code can be set to empty